docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
 
-  \begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2
-  \end{align*}
+The population-attributable fractions (PAFs) must be calculated correctly,
+or the population-level incidence of the outcome will
+not validate to GBD.
+**The usual Vivarium strategy to calculate joint PAFs (PAFs of more than one risk factor on an outcome)
+is biased when the risk factors are correlated.**
 
-Dividing through and cancelling terms yields a solution:
+The true population attributable fraction for the combined effect of two risk factors,
+:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is (assuming no effect modification, as we typically do):
 
 .. math::
-
-
-   \frac{1}{1-\text{PAF}} = \int_{e_1} \int_{e_2} \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-   p_{e_1,e_2} de_1 de_2,
-
-.. code:: python
-
-    # there are lots of ways to approximate this integral, but here is a cool one:
-    rr_1 = 1.5
-    rr_2 = 3
-
-    e1 = np.clip((df['SBP'] - 120)/10, 0, np.inf)  # rescale exposures
-    e2 = np.clip((df['LDL-C'] - 4), 0, np.inf)
-
-    one_over_one_minus_paf = np.mean(rr_1**e1 * rr_2**e2)
-    paf = 1 - 1/(one_over_one_minus_paf)
-    paf
-
-
-
-
-.. parsed-literal::
-
-    0.936944660107624
-
-
-
-.. code:: python
-
-    # how does this compare to multiplicative approx of paf?
-    ooomp_1 = np.mean(rr_1**e1)
-    paf_1 = 1 - 1/ooomp_1
-
-    ooomp_2 = np.mean(rr_2**e2)
-    paf_2 = 1 - 1/ooomp_2
-
-    1 - (1 - paf_1) * (1 - paf_2)
-
-
-
-
-.. parsed-literal::
-
-    0.8708098034918508
-
-
-
-These values are somewhat different, reflecting the impact of the
-correlation in SBP and LDL-C in our simulated data. If we used the
-*multiplicative* PAF, our estimated rate would be quite different. Let’s
-look at a simpler model of categorical risks.
-
-Here is an example of categorial risk exposures
-=======================================================
-
-.. code:: python
-
-    # simulate categorical data (to be replaced with real data, e.g. from MALED/Surveys, eventually)
-    df = pd.DataFrame(index=range(N))
-    # Uncorrelated, binomial
-    df['bin_a'] = np.random.binomial(1, 0.3, size=N)
-    df['bin_b'] = np.random.binomial(1, 0.1, size=N)
-
-    rr_1 = 2.5
-    rr_2 = 5
-
-
-
-Define what we will call a joint PAF:
+  O = O_{tmrel} * RR_{r1} * RR_{r2}
 
 .. math::
+  PAF = \frac{E(O) - E(O_{tmrel})}{E(O)} = 1 - \frac{O_{tmrel}}{E(O_{tmrel} * RR_{r1} * RR_{r2})} = 1 - \frac{1}{E(RR_{r1} * RR_{r2})}
 
+where :math:`O_{tmrel}` is a constant representing the outcome (rate) among those at the
+theoretical minimum risk exposure level.
```

Comment:
TMREL across the joint distribution of r1 and r2 here.

Reply:
@alibow Is it not the case that the TMRED on the joint distribution is always where r1 = TMREL(r1) and r2 = TMREL(r2) for all individuals?

Reply:
Well, I wouldn't be surprised if that's what GBD actually does, but in general, I don't think there's any reason that this should always be true (I'd have to think about it more to be sure).

Reply:
But for the only pair of risks that GBD actually models jointly, _(birth weight, gestational age) = Low Birth Weight and Short Gestation risk factor_, they first define the TMREL for the joint (bw, ga) distribution, then they have some confusing and conceptually suspicious process of trying to identify separate univariate TMRELs for BW and GA. I think their strategy for identifying the univariate TMRELs changed from GBD 2017 to GBD 2019, and I'm pretty sure that in either case, the univariate TMRELs don't  necessarily coincide with the projections of the joint TMREL onto the two components.

Reply:
I am not sure I follow the LBWSG example, but I'd expect that to be the one _exception_ since that is the one case where they use sophisticated methods to get a joint PAF. In the usual case, they use the multiplicative approach `PAF_joint = 1 - (1 - PAF_1) * (1 - PAF_2)`. This would seem to make no sense at all if the joint TMREL was unrelated to the univariate TMRELs.

Reply:
Oh, but I guess if we're assuming RR(r1,r2) = RR(r1)*RR(r2) and RR's are >=1, then the points where RR(r1,r2) = 1 are precisely the points (r1,r2) where RR(r1)=1 and RR(r2)=1.

So maybe it would be good to explicitly say that the joint TMREL coincides with the Cartesian product of the univariate TMRELs because of our assumption about the form of the joint relative risk.

Reply:
> I am not sure I follow the LBWSG example, but I'd expect that to be the one _exception_ since that is the one case where they use sophisticated methods to get a joint PAF.

The (BW, GA) coordinates are binned into 58 categories (give or take), and then they estimate the RR for each LBWSG exposure category, just like any other categorical risk. In particular, the relative risks do **not** satisfy the assumption RR(BW, GA) = RR(BW)*RR(GA) for the joint risk.

Reply:
Yeah, I just meant that as written, the TMREL here should be RR_r1=1 and RR_r2=1, and we typically think of TMRELs in terms of a single risk exposure, so I just thought we could clarify for this situation that we mean for both risks.